src/test/java/org/example/c20debugging/UtilitiesTestParameterized.java

```diff
@@ -0,0 +1,41 @@
+package org.example.c20debugging;
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.Arrays;
+import java.util.Collection;
+
+@RunWith(Parameterized.class)
```

Comment:
Use Junit5 version of parametrized tests. See https://www.baeldung.com/parameterized-tests-junit-5

Reply:
done